pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta:__legacy__"
+
+[project]
+name = "jep"
+description = "Jep embeds CPython in Java through JNI."
+dynamic = ["version"]
+#version = "4.3.0"
```

Comment:
Why is version commented out?

Reply:
I forgot to remove that when I got the dynamic version working. Leaving the version specifier in the setup() call in setup.py allows it to pick up the version in version.py so we don't have to remember to keep this version in sync. I will remove the commented out version.